server/routerlicious/packages/services/src/rdkafkaProducer.ts

```diff
@@ -67,7 +68,7 @@ export class RdkafkaProducer extends RdkafkaBase implements IProducer {
 			...this.producerOptions.additionalOptions,
 		};
 
-		this.producer = new kafka.HighLevelProducer(options);
+		this.producer = new kafka.HighLevelProducer(options, this.producerOptions.topicConfig);
```

Comment:
Where are these values defined? Or these are optional values and HighLevelProducer fills them out if not present?

Reply:
They are optional. The defaults are listed [here](https://github.com/edenhill/librdkafka/blob/fc7e0ee60155079cff6b553211906c01786523b8/CONFIGURATION.md), under the producer topic config section.